sips/sip-112.md

```diff
@@ -71,9 +73,18 @@ interface INativeEtherWrapper {
     function burn(uint amount) external;
 }
 ```
-
+#### L2 Specification
 `NativeEtherWrapper` is not deployed for L2, as using native ETH opcodes (`callvalue`, `balance`) in the OVM will revert.
 
+#### Key Bounds
+
+- The upper bound on the amount of  *Minting* is determined with a helper function, `capacity`, defined as `maxETH` less `wETH` locked in the contract. In case the user attempts to mint an amount greater than the upper bound, then `capacity` is minted and the residual is returned to the user (please refer to test cases for calculation specs).
```

Comment:
nitpick - let's use consistent naming for wETH or WETH (I prefer the latter).

Reply:
old habits die hard, 😅  all `wETH` changed to `WETH`